preview_generator/preview/builder/document_generic.py

```diff
@@ -142,20 +142,17 @@ def build_pdf_preview(
         if page_id < 0:
             return  # in this case, the intermediate file is the requested one
 
-        pdf_in = utils.get_decrypted_pdf(intermediate_pdf_file_path)
```

Comment:
I don't see how this worked before, as `get_decrypted_pdf` calls `seek(0)` on its first parameter...